packages/recipe/src/components/EzTable/EzTable.tsx

```diff
@@ -546,7 +547,7 @@ const EzTable: FC<TableProps> = ({
   if (!title) return table;
 
   return (
-    <EzCard title={title} subtitle={subtitle} actions={actions}>
+    <EzCard title={title} subtitle={subtitle} actions={actions} titleIcon={titleIcon || null}>
```

Comment:
I think we can leave out the `|| null` here as it will be `undefined` if not provided.
```suggestion
    <EzCard title={title} subtitle={subtitle} actions={actions} titleIcon={titleIcon}>
```